File: php/Mhsiswa.php
<?php

/*Saya Muhammad Daffa Yusuf Fadhilah dengan NIM 2100543 mengerjakan evaluasi Latihan 1
dalam mata kuliah Design Pemrograman Berorientasi Objek 2023
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti
yang telah dispesifikasikan. Aamiin. */

//Pembuatan clas Mhsiswa
class Mhsiswa
{

    //PEmbuatan attribut yang dibutuhkan
    private $nama = '';
    private $nim = '';
    private $prodi = '';
    private $fakultas = '';

    //Construcot yang dibuat kosong untuk Mhsiswa
    public function __construct($nama = '', $nim = '', $prodi='', $fakultas=''){
        $this->nama = $nama;
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
    }

    
    //Setter dan getter tipa atribut
    public function set_nama($nama){
        $this->nama= $nama;
    }

    public function set_nim($nim){
        $this->nim = $nim;
    }

    public function set_prodi($prodi){
        $this->prodi = $prodi;
    }

    public function set_fakultas($fakultas){
        $this->fakultas = $fakultas;
    }

    public function get_nama(){
        return $this->nama;
    }

    public function get_nim(){
        return $this->nim;
    }

    public function get_prodi(){
        return $this->prodi;
    }

    public function get_fakultas(){
        return $this->fakultas;
    }
}
?>

File: Python/Mhsiswa.py
# Saya Muhammad Daffa Yusuf Fadhilah dengan NIM 2100543 mengerjakan evaluasi Latihan 1
# dalam mata kuliah Design Pemrograman Berorientasi Objek 2023
# untuk keberkahanNya maka saya tidak melakukan kecurangan seperti
# yang telah dispesifikasikan. Aamiin.

# Pembuatan clas Mhsiswa
class Mhsiswa:
    
    # PEmbuatan attribut yang dibutuhkan
    __nama = ""
    __nim = ""
    __prodi = ""
    __fakultas = ""
    
    # Construcot yang dibuat kosong untuk Mhsiswa
    def __init__(self, nama="", nim="", prodi="", fakultas= ""):
        self.__nama = nama
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas
    
    # etter dan getter tipa atribut    
    def set_nama(self, nama):
        self.__nama = nama
    
    def set_nim(self, nim):
        self.__nim = nim
    
    def set_prodi(self, prodi):
        self.__prodi = prodi
    
    def set_fakultas(self, fakultas):
        self.__fakultas = fakultas
    
    def get_nama(self):
        return self.__nama
    
    def get_nim(self):
        return self.__nim
    
    def get_prodi(self):
        return self.__prodi
    
    def get_fakultas(self):
        return self.__fakultas